src/main/resources/templates/error.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html lang="cs" xmlns:th="http://www.thymeleaf.org" style="height:100%">
+<head th:replace="fragments/head :: head('Luštěniny - chyba')">
+</head>
+<body class="login-page-body">
+<div class="container login-form-container">
+    <div class="row">
+        <div class="col-sm-12">
+            <h2 class="login-form-header">Chyba, tady nic nenajdeš...</h2>
+            <a href="#" class="btn btn-primary" th:href="@{/successfulLogin}">Zpět</a>
```

Comment:
Asi bych to nepojmenovával jako zpět, když to uživatele opravdu nevezme zpět.

Reply:
a jak?

Reply:
Třeba Domů, Jít domů, Homepage, něco takového.

Reply:
zmeneno na Domů